src/org/exist/util/serializer/AbstractSerializer.java

```diff
@@ -30,7 +30,7 @@
         new HTML5Writer()
     };
 
-    private final static Properties defaultProperties = new Properties();
+    protected final static Properties defaultProperties = new Properties();
```

Comment:
I am wondering if the fact that this is static would cause concurrent serializations to screw with each others properties? e.g. `AdaptiveSerialization#setOutput` I think could be called by concurrent XQueries?